### ISSUE TITLE
CVEs on 1.9.1

### ISSUE THREAD

**Issue by tooptoop4** [GoogleContainerTools/kaniko#2276]
/kaniko/docker-credential-gcr has github.com/docker/docker (v20.10.8) affected by CVE-2021-39293,CVE-2021-41089,CVE-2021-41091,CVE-2021-41103,CVE-2022-1705,CVE-2022-1962,CVE-2022-24769,CVE-2022-27664,CVE-2022-28131,CVE-2022-29162,CVE-2022-30630,CVE-2022-30631,CVE-2022-30632,CVE-2022-30633,CVE-2022-30635,CVE-2022-32148,CVE-2022-32189,CVE-2022-32190,CVE-2022-36109

/kaniko/docker-credential-gcr has github.com/docker/cli (v20.10.8) affected by CVE-2021-39293, CVE-2021-41092

/kaniko/docker-credential-acr-env has golang.org/x/crypto (0.0.0-20201002170205-7f63de1d35b0) affected by CVE-2020-9283, CVE-2022-27191

/kaniko/docker-credential-acr-env has github.com/dgrijalva/jwt-go (3.2.0) affected by CVE-2020-26160

**Comment by liam-verta** [comment]
This kind of blanket reporting of CVEs is not useful - it makes it very hard to track progress and know when the issues are finally closed.

It looks like most of these are closed. 

In v1.9.1, the first three items are addressed:
* Fixed: github.com/docker/docker has been updated to v20.10.14 - [go.mod#L20](https://github.com/GoogleContainerTools/kaniko/blob/f9aaa9fca7bf4077778ed527c1a8a6e09e60c53c/go.mod#L20)
* Fixed: github.com/docker/docker has been updated to v20.10.12 - [go.mod#L88](https://github.com/GoogleContainerTools/kaniko/blob/f9aaa9fca7bf4077778ed527c1a8a6e09e60c53c/go.mod#L88)
* Fixed: golang.org/x/crypto has been updated to v0.0.0-20220112180741-5e0467b6c7ce - [go.mod#L141](https://github.com/GoogleContainerTools/kaniko/blob/f9aaa9fca7bf4077778ed527c1a8a6e09e60c53c/go.mod#L141)

Usually, `go list -m all`will get a list of the actual version of each dependency that is used.  Kaniko "vendors" all their dependencies, so we have to use `go list -m all -mod=mod` to get that list.  It confirms that the above items are fixed. 

The four item,`github.com/dgrijalva/jwt-go` is confusing.  It is not present in the `vendor` directory, but it can be found in the `go list` output. The same with `spf13/viper` which appears to be a common transitive source of `jwt-go`.  I'm not sure why this is the case. I'll open a new issue to track this down.

**Comment by aaron-prindle** [comment]
This is resolved now with the release today of `v1.12.0`:

```
aprindle@aprindle-ssd ~/kaniko  [main]grype gcr.io/kaniko-project/executor:v1.12.0
 ✔ Vulnerability DB        [no update available]
New version of grype is available: 0.63.0 (currently running: 0.61.0)
 ✔ Loaded image            
 ✔ Parsed image            
 ✔ Cataloged packages      [202 packages]
 ✔ Scanning image...       [2 vulnerabilities]
   ├── 0 critical, 1 high, 1 medium, 0 low, 0 negligible
   └── 0 fixed
No vulnerabilities found
```

Closing
